feat(sdk): expose insecure/ca-certs registry overrides in Go and Python (#1247)
superradcompany/microsandbox: docs/sdk/python/sandbox.mdx

| pull_policy | `str \| `[`PullPolicy`](#pullpolicy) | `"if-missing"` | Image pull behavior |
| log_level | `str \| `[`LogLevel`](#loglevel) | - | Override log verbosity |
| registry_auth | [`RegistryAuth`](#registryauth) | - | Private registry credentials |
| registry_insecure | `bool` | `False` | Pull images over plain HTTP instead of HTTPS (local or self-hosted registries) |

> | registry_ca_certs | `list[bytes \| str \| os.PathLike]` | `[]` | Extra CA roots to trust when pulling. Each entry is either PEM data (`bytes`/`bytearray`) or the path of a PEM file, read when `create()` is called |

Reviewer: ```suggestion
| registry_ca_certs | `list[bytes \| bytearray \| str \| os.PathLike]` | `[]` | Additional CA roots for image pulls. File paths are read when `create()` is called. |
```
